lib/Telemetry/src/SensorVoltage.cpp

```diff
@@ -0,0 +1,25 @@
+#include "SensorVoltage.h"
+
+#define INPUT_VOLTAGE_PIN A1
+#define ANALOG_CONVERT_RAW_TO_3V 1241
```

Comment:
Just tested this at home with my bench-top power supply and multi-meter. Looks like 1230 is the closest value, I did a bit of trial and error. 

Can you also move these defines into the header (.h) file? There's no functional reason for it, just to match the style in the rest of the project so far.